app/controllers/sessions_controller.rb

```diff
@@ -16,6 +16,14 @@ def create
 
   def destroy
     session[:user_id] = nil
+    session[:github_token] = nil
     redirect_to root_path
   end
+
+  def update
+    github_token = request.env['omniauth.auth'][:credentials][:token]
+    current_user.update(github_token: github_token)
+    session[:github_token] = github_token if current_user.save
```

Comment:
Thinking we should store the `github_token` in a session, and update user for `github_handle`

Reply:
I was having trouble getting it to update the user in the database using `current_user` (the error said something about password required) so ended up using `update_attribute` instead and actually looking up the User object using session[:email]. I tried this code locally with rails server and couldn't get it to work. Very probably user error, but just verifying that this actually updates the database.